### PR TITLE
Adding Pt binned QCD samples for PHYS14

### PR DESCRIPTION
**Pull request by artlbv**
Some samples not available on EOS.

### CODE REVIEW

**Comment by gpetruc** [comment]
Could you add the cross sections, and update the PR?

**Comment by artlbv** [comment]
Somehow I can't find them anywhere for these samples. 
Neither DAS shows nothing, nor MCM:
https://cms-pdmv.cern.ch/mcm/requests?produce=%2FQCD_Pt-120to170_Tune4C_13TeV_pythia8%2FPhys14DR-PU20bx25_trkalmb_castor_PHYS14_25_V1-v2%2FMINIAODSIM&page=0&shown=786559

Do you have any idea where else they could be given?

p.s. the QCD_Pt samples in CSA14.py also don't have any cross-sections, unfortunately.

**Comment by artlbv** [comment]
Alright, I found them. 
I'll update the PR and let you know.

**Comment by gpetruc** [comment]
Usually they're in the GEN-SIM on McM

```
 Giovanni
```

Il 10-feb-2015 14:37 "Artur Lobanov" notifications@github.com ha scritto:

**Comment by artlbv** [comment]
Done, xsecs now in the same branch.

File: CMGTools/TTHAnalysis/python/samples/samples_13TeV_PHYS14.py [hunk 1]
```diff
@@ -104,6 +104,40 @@
 QCD_HT_1000ToInf_ext1
 ]
 
+QCD_Pt15to30 = kreator.makeMCComponent("QCD_Pt15to30","/QCD_Pt-15to30_Tune4C_13TeV_pythia8/Phys14DR-PU20bx25_trkalmb_castor_PHYS14_25_V1-v1/MINIAODSIM", "CMS", ".*root", 2237000000)
+QCD_Pt30to50 = kreator.makeMCComponent("QCD_Pt30to50","/QCD_Pt-30to50_Tune4C_13TeV_pythia8/Phys14DR-PU20bx25_trkalmb_castor_PHYS14_25_V1-v2/MINIAODSIM", "CMS", ".*root", 161500000)
```
Comment (gpetruc) [inline CMGTools/TTHAnalysis/python/samples/samples_13TeV_PHYS14.py]: It seems to me `/QCD_Pt-30to50_Tune4C_13TeV_pythia8/Phys14DR-PU20bx25_trkalmb_castor_PHYS14_25_V1-v2/MINIAODSIM` does not exist in DAS, while `/QCD_Pt-30to50_Tune4C_13TeV_pythia8/Phys14DR-PU20bx25_trkalmb_castor_PHYS14_25_V1-v1/MINIAODSIM` does. 

Can you check, and if needed fix and update this PR?

**Comment by artlbv** [comment]
You're right. I somehow missed this one while running.
I've also set some samples to use AAA, since they are not available on EOS.

**Pull request merged**